.travis.yml

```diff
@@ -49,8 +43,5 @@ deploy:
   provider: releases
```

Comment:
Since we don't deploy anything anymore, it seems like deploy section can go away entirely.

Reply:
Done.